js/components/braveryPanel.js

```diff
@@ -0,0 +1,240 @@
+          </div>
+          <div onClick={this.onToggleTPList}>
+            <div className='braveryStat trackersBlockedStat'>{this.blockedByTrackingList ? this.blockedByTrackingList.size : 0}</div>
+            <div data-l10n-id='trackersBlocked' />
+          </div>
+          <div onClick={this.onToggleHttpseList}>
+            <div className='braveryStat redirectedResourcesStat'>{this.redirectedResources ? this.redirectedResources.size : 0}</div>
+            <div data-l10n-id='httpReroutes' />
+          </div>
+        </div>
+        <div className='braveryPanelBody'>
+          <ul>
+          {
+            this.isTPListShown && this.blockedByTrackingList && this.blockedByTrackingList.size > 0
+            ? <li><ul>
+            {
+              this.blockedByTrackingList.map((site) => <li key={site}>{site}</li>)
+            }
+            </ul></li>
+            : null
+          }
+          {
+            this.isBlockingAds && this.isBlockedAdsShown
+            ? <li><ul>
+            {
+              this.blockedAds.map((site) => <li key={site}>{site}</li>)
+            }
+            </ul></li>
+            : null
+          }
+          {
+            this.isRedirectingResources && this.isHttpseShown
+            ? <li><ul>
+            {
+              this.redirectedResources.map((sites, ruleset) =>
+                <li key={ruleset}>{[ruleset, JSON.stringify(sites.toJS())].join(': ')}</li>)
+            }
+            </ul></li>
+            : null
+          }
+          </ul>
+          <div className={cx({
+            braveryAdvancedTitle: true,
+            disabled: !shieldsUp
+          })} onClick={this.onToggleAdvanced}>
+            <div className={cx({
+              fa: true,
+              'fa-caret-down': this.isAdvancedExpanded,
+              'fa-caret-right': !this.isAdvancedExpanded,
+              braveryAdvancedIndicator: true
+            })} />
+            <div data-l10n-id='advancedControls' />
+          </div>
+          {
+            this.isAdvancedExpanded
+            ? <span>
+              <hr />
+              <div className='braveryAdvanced'>
+                <div className='braveryControlGroup'>
+                  <div className={cx({
+                    braverySelectTitle: true,
+                    disabled: !shieldsUp
+                  })} data-l10n-id='adControl' />
+                  <select value={this.getSiteSetting('adControl', 'showBraveAds')} onChange={this.onToggleAdControl} disabled={!shieldsUp}>
+                    <option data-l10n-id='showBraveAds' value='showBraveAds' />
+                    <option data-l10n-id='blockAds' value='blockAds' />
+                    <option data-l10n-id='allowAdsAndTracking' value='allowAdsAndTracking' />
+                  </select>
+                  <SwitchControl onClick={this.onToggleSafeBrowsing} rightl10nId='safeBrowsing' checkedOn={this.getSiteSetting('safeBrowsing', true)} disabled={!shieldsUp} />
```

Comment:
I would move the safebrowsing switch last, since it's the one that is least useful